.github/workflows/build-book.yml

```diff
@@ -44,6 +44,7 @@ jobs:
         directory: '_build'
         arguments: |
            --ignore-files "/.+\/_static/"
+           --ignore-url "/html/_static/webpack-macros.html"
```

Comment:
frustrated because this was working and now it's not. ignore files worked in another repo to ignore that entire directory.